### PR TITLE
Reset timer at 9am

### PR DESCRIPTION
**Pull request by krestkrest**
Reset timer at 9am to prevent the time window slipping every day during the working week.
Also upgraded jest version for security reasons (npm asked me to do that).

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -34,4 +34,38 @@ let game = new Game(adapter, repository, gamesRepository);
...
+test('next day after 9am', () => {
+    let now = new Date(2021, 1, 2, 9, 0, 1);
+    expect(game.canStartGame(prev, now)).toBe(true);
+})
+
+test('same day', () => {
+    let prev = gameWithDate(2021, 1, 2, 1, 0, 1);
+    let now = new Date(2021, 1, 2, 10, 0, 1);
+    expect(game.canStartGame(prev, now)).toBe(false);
+})
```
Comment (surranov) [inline test.js]: new line at end of line

File: src/Game.js [hunk 2]
```diff
@@ -103,13 +104,24 @@ class Game {
...
+
+        if (prev_game.datetime <= Math.floor(now / 1000) - 86400)
+            return true;
+
+        let prevGameDate = new Date(prev_game.datetime * 1000);
+        return prevGameDate.getDate() !== now.getDate() && now.getHours() >= 9;
+    }
 }
 
 module.exports = Game;
```
Comment (surranov) [inline src/Game.js]: new line at end of line

**Review by surranov — CHANGES_REQUESTED** [review]

File: test.js (hunk 1 above)
Comment (krestkrest) [reply test.js]: fixed

File: src/Game.js (hunk 2 above)
Comment (krestkrest) [reply src/Game.js]: fixed

**Review by surranov — APPROVED** [review]

**Comment by ecrofeg** [comment]
Огонь бля

**Pull request merged**